extensions/cache/deployment/src/main/java/io/quarkus/cache/deployment/devconsole/CacheDevUiConsoleProcessor.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.cache.deployment.devconsole;
+
+import io.quarkus.cache.runtime.devconsole.CacheJsonRPCService;
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.pkg.builditem.CurateOutcomeBuildItem;
+import io.quarkus.devui.spi.JsonRPCProvidersBuildItem;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.Page;
+
+public class CacheDevUiConsoleProcessor {
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    CardPageBuildItem create(CurateOutcomeBuildItem bi) {
+        CardPageBuildItem pageBuildItem = new CardPageBuildItem("Cache");
+        pageBuildItem.addPage(Page.webComponentPageBuilder()
+                .title("Caches")
+                .componentLink("cache-component.js")
```

Comment:
This needs to be qwc-cache-caches. This takes the format of, qwc-extensionName-pageName